src/lib/journey/callbacks/password/confirm-input.svelte

```diff
@@ -24,14 +25,26 @@
 
   let isVisible = false;
   let type: 'password' | 'text' = 'password';
-  let value: unknown;
+  let value: Maybe<string>;
+
+  function onChangeWrapper(event: Event) {
```

Comment:
Creating a wrapper here to grab the value of the input before passing it to the parent `onChange` function.